drivers/SmartThings/jbl/src/lunchbox/rest.lua

```diff
@@ -158,7 +190,9 @@ local function handle_response(sock)
   if initial_recv ~= nil then
     local headers = initial_recv:get_headers()
 
-    if headers and headers:get_one("Transfer-Encoding") == "chunked" then
+    if headers:get_one("Content-Length") then
```

Comment:
I think this will be a problem if the headers are empty, and it should be `if headers and headers:get_one` like the one below.

Reply:
This also seems like it would never go to the `elseif` below.  Was it ever the case that the `Content-Length` header wasn't set, but the `Transfer-Encoding` was?